test/jdk/java/security/Security/ClassLoaderDeadlock/ClassLoaderDeadlock.java

```diff
@@ -21,7 +21,15 @@
  * questions.
  */
 
-// See bug 5094825 / ClassLoadDeadlock.sh
+/*
+* @test
+* @bug 5094825
+* @summary verify no deadlock if crypto provider in other classloader is used to verify signed jars
+* @modules java.base/java.security
```

Comment:
Hello Mikhail, `java.security` package is an exported package from `java.base`. So you don't need this `@modules` declaration.